From: Chow Loong Jin <hyperair@debian.org>
Date: Sun, 7 Nov 2021 12:18:23 +0800
Subject: Don't disable visibility on ppc64 arch

On ppc64, builds were failing because there are too many extra symbols being
exported in the library as a result of --enable-visibility being forced off.

Bug-Debian: https://bugs.debian.org/825038
---
 Project/GNU/Library/configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Project/GNU/Library/configure.ac b/Project/GNU/Library/configure.ac
index 024f9da..d8135ba 100644
--- a/Project/GNU/Library/configure.ac
+++ b/Project/GNU/Library/configure.ac
@@ -803,9 +803,6 @@ case "${build}" in
     CFLAGS="$CFLAGS -D__MONTAVISTA__"
     CXXFLAGS="$CXXFLAGS -D__MONTAVISTA__"
   ;;
-  powerpc64-* )
-    enable_visibility="no"
-  ;;
 esac
 
 dnl -------------------------------------------------------------------------
